tensorflow/python/ops/sampling_ops.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import numpy as np
+
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import tensor_shape
+from tensorflow.python.framework import tensor_util
+# pylint: disable=wildcard-import
+# 'Constant' gets imported in the module 'array_ops'.
+from tensorflow.python.ops import gen_sampling_ops
```

Comment:
also:

`from tensorflow.python.ops.gen_sampling_ops import *`

any non-hidden ops should be imported directly as well.